.github/workflows/container.yml

```diff
@@ -47,6 +47,7 @@ jobs:
           file: ./Dockerfile
           build-args: |
             KANIKO_VERSION=1.23.2
+            ASSUME_ROLE_VERSION=1.3.9
```

Comment:
Wouldn't it be better to name the variable after the source `COOKIELAB_DEPLOYER_VERISON`? Or in case we'd like to have multiple scripts from different versions of deployer image `COOKIELAB_DEPLOYER_ASSUME_ROLE_VERSION`? Yes, it's long, but clear what it means on the first sight.